rebase-blogpost/README.md

```diff
@@ -44,6 +45,7 @@ This problem is greater than it may seem at first.
 Why do we use Git at all? Because it is our most important tool for tracking down the source of bugs in our code. Git is our safety net. By rebasing, we give this less priority, in favour of the desire to achieve a linear history.
 
 This is not only a meaningless prioritization. It is bad craftsmanship, and irresponsible towards the rest of our team.
+> Aslak: The bad craftmanship argument is weak - what constitutes good/bad craftmanship is very subjective. I would focus only on the strongest argument: The ability to quickly diagnose and fix a bug.
```

Comment:
I agree. I'm tending towards just removing it and leaving the previous paragraph unchanged.